### ISSUE TITLE
[feature] Add phone number (if there is)

### ISSUE THREAD

**Issue by Azecko** [epfl-si/ATARI#78]
We need to display phone number for users that have one (or many)

**Comment by Azecko** [comment]
`/persons/` route → take the first two phone numbers (if there is) and display them

**Comment by vigarov** [comment]
Hello,
Is there an update on this? 

Victor

**Comment by vigarov** [comment]
No worries! I can't seem to find the issue you mention, but if the solution you had in mind was to show the 2 first phone numbers of a person, then it sounds good to me!

**Comment by vigarov** [comment]
Ping..

**Comment by Azecko** [comment]
Hi Victor,

First, I apologize for the insane delay on this issue.
Normally, phone numbers should now be displayed under the username. It's ready to be tested on https://atari-test.epfl.ch/

Can you please try and tell me if it works as you wish ?

**Comment by vigarov** [comment]
The login on test doesn't seem to work (nothing happens when I press the button on top right, or the login text inside the main paragraph; tested on Firefox and chrome in private windows), I can't try the feature.

**Comment by Azecko** [comment]
You are right, looks like authentication is broken on test. Working on it.
